res/controllers/Pioneer-DDJ-200-scripts.js

```diff
@@ -181,7 +181,7 @@ DDJ200.headmix = function(channel, control, value) {
     if (value) { // do nothing if button is released, i.e. value === 0
         var masterMixEnabled = (engine.getValue("[Master]", "headMix") > 0);
         engine.setValue("[Master]", "headMix", masterMixEnabled ? -1 : 1);
-        midi.sendShortMsg(0x96, 0x63, masterMixEnabled ? 0x7F : 0); // set LED
+        midi.sendShortMsg(0x96, 0x63, masterMixEnabled ? 0 : 0x7F); // set LED
```

Comment:
Which LED is controlled? A comment might help to understand why this LED needs to reflect the inverse(!) state of masterMixEnabled. This is not obvious from the magic numbers.

Reply:
I assume its the button labeled "Master" in the middle of the controller. Its weird that this one seems to be inverted but the 1 and 2 buttons below it are not...

Reply:
Not only fix the bug but always think about how it could have been prevented and how to prevent any future regressions.

Reply:
The following happens:
- `masterMixEnabled` get the actual value from mixxx
- `engine.setValue ` sets the inverse value
- `midi.sendShortMsg` was sending the led signal from masterMixEnabled, before been inverted. (This is what my code fixes)

I will rewrite this peace of code to be clear.

Reply:
I'll merge this PR now to get the fix in asap. Improvements could be done in a follow-up PR.